ledger/object/index.go

```diff
@@ -31,6 +31,8 @@ import (
 type IndexModifier interface {
 	// SetIndex adds a bucket with provided pulseNumber and ID
 	SetIndex(ctx context.Context, pn insolar.PulseNumber, index record.Index) error
+	// UpdateLastKnownPulse updates last know pulse to given one for all objects from this pulse
+	UpdateLastKnownPulse(ctx context.Context, pn insolar.PulseNumber) error
```

Comment:
why it's here? don't abuse interface please

Reply:
We should probably create another interface for that.

Reply:
Interface is called IndexModifier. This method suits to that. No?